web/handlers/admin/invites_test.go

```diff
@@ -143,17 +144,29 @@ func TestInvitesCreate(t *testing.T) {
 	a := assert.New(t)
 	r := require.New(t)
 
-	urlRemove := ts.URLTo(router.AdminInvitesCreate)
+	urlCreate := ts.URLTo(router.AdminInvitesCreate)
 
 	testInvite := "your-fake-test-invite"
 	ts.InvitesDB.CreateReturns(testInvite, nil)
 
-	rec := ts.Client.PostForm(urlRemove, url.Values{})
-	a.Equal(http.StatusOK, rec.Code)
+	totalCreateCallCount := 0
+	createInviteShouldWork := func(works bool) *httptest.ResponseRecorder {
+		rec := ts.Client.PostForm(urlCreate, url.Values{})
+		if works {
+			totalCreateCallCount += 1
+			a.Equal(http.StatusOK, rec.Code)
+			r.Equal(totalCreateCallCount, ts.InvitesDB.CreateCallCount())
+			_, userID := ts.InvitesDB.CreateArgsForCall(totalCreateCallCount - 1)
+			a.EqualValues(ts.User.ID, userID)
+		} else {
+			// TODO: status should be http.StatusForbidden? see invites.go:79
```

Comment:
@cryptix any chiming on this? it had me stumped

Reply:
resolved by https://github.com/ssb-ngi-pointer/go-ssb-room/pull/200/commits/41b9de268758e1349aabc1edb5a22c847e41c836, thanks again!